lib/manageiq/api/client/action.rb

```diff
@@ -0,0 +1,17 @@
+module ManageIQ
+  module API
+    module Client
+      class Action
+        attr_accessor :name
+        attr_accessor :method
+        attr_accessor :href
+
+        def initialize(action_hash)
```

Comment:
wanted to use this, but name:, method:, ... params does not support string keys, just symbols, so didn't work well with the json response coming from the server.